src/components/DateRangePickerInput.jsx

```diff
@@ -123,6 +128,22 @@ export default class DateRangePickerInput extends React.Component {
     });
   }
 
+  renderInputIcon(calendarIcon) {
```

Comment:
I'd rather have this as a variable `inputIcon` in the `render` method than its own method.